### PR TITLE
Add Basic page structure

### PR DESCRIPTION
**Pull request by SurafelGetachewT**
In this project I have done the following:

1. Create an npm project with webpack.
2. Write plain HTML markup with minimum styling (just to make the layout) using the given wireframe as a guide.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
-# LeaderBoard
\ No newline at end of file
+# LeaderBoard
+
```
Comment (HAMMAS-SALEEM) [inline README.md:2]: - [ ] Please add a short description about your project here to make your readme descriptive :blush:

File: README.md [hunk 2]
```diff
@@ -1 +1,55 @@
...
+Feel free to check the [issues page](https://github.com/SurafelGetachewT/LeaderBoard/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](LICENCE.md) licensed.
```
Comment (HAMMAS-SALEEM) [inline README.md:55]: - [ ] Please fix your link to your LICENSE file. Replace `[MIT](LICENSE.md)` to `[MIT](./LICENSE)` :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @SurafelGetachewT :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Meaningful commits
:heavy_check_mark: Added .gitignore
:heavy_check_mark: Following GitFlow

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional Suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. Otherwise reach out to me on slack which is more fast. You can reach out to me on slack by searching my name in **full-time-students** channel :+1:

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @SurafelGetachewT, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**